apps/backoffice-v2/src/pages/Entity/components/Case/hooks/useCaseActionsLogic/useCaseActionsLogic.tsx

```diff
@@ -80,6 +80,7 @@ export const useCaseActionsLogic = ({ workflowId, fullName }: IUseActions) => {
     ? {
         id: workflow?.assignee?.id,
         fullName: `${workflow?.assignee?.firstName} ${workflow?.assignee?.lastName}`,
+        avatarUrl: workflow?.assignee?.avatarUrl as string | null,
```

Comment:
Use `?? ''`, `?? null`, or cover the undefined case here instead. The type should be right from the schema in `fetchers.ts` unless `optional` is not true.